include/uapi/sound/sof/tokens.h

```diff
@@ -153,6 +153,8 @@
 /* Stream */
 #define SOF_TKN_STREAM_PLAYBACK_COMPATIBLE_D0I3	1200
 #define SOF_TKN_STREAM_CAPTURE_COMPATIBLE_D0I3	1201
+#define SOF_TKN_STREAM_PLAYBACK_PAUSE_SUPPORTED	1202
+#define SOF_TKN_STREAM_CAPTURE_PAUSE_SUPPORTED	1203
```

Comment:
commit message:
SOF_TKN_STREAM_PLAYBACK_CAPTURE_SUPPORTED ->  SOF_TKN_STREAM_CAPTURE_PAUSE_SUPPORTED

Reply:
gahs, thanks for catching it.